packages/dds/tree/CHANGELOG.md

```diff
@@ -1,5 +1,60 @@
 # @fluidframework/tree
 
+## 2.12.0
```

Comment:
I guess the changeset said this change affected tree but I don't think it does...? This content probably shouldn't be here for this package, I think.

Reply:
No. We upgrade this version in all packages.

Reply:
Sorry, the 2.12.0 section yes, I meant the content inside it.

Reply:
"@fluidframework/azure-client": minor
"@fluidframework/container-definitions": minor
"@fluidframework/container-loader": minor
"@fluidframework/fluid-runner": minor
"@fluidframework/odsp-client": minor
"@fluid-experimental/property-dds": minor
"@fluid-private/test-end-to-end-tests": minor
"@fluidframework/test-utils": minor
"@fluidframework/tinylicious-client": minor
"@fluidframework/tree": minor

I think what it does is it looks at the top section of the file and put the above changes in all packages mentioned here.

Reply:
So maybe I should remove from other packages and just keep it in container loader.

Reply:
Yeah, the changeset would have been fine to only specify container-loader IMO. By default `pnpm changeset` will select all packages where there are changed files, but the real API change is just in container-loader (in this case).